ChangeLog

```diff
@@ -122,6 +122,10 @@ What's New in Pylint 1.8?
       mixing ``Args`` and ``Keyword Args`` in Google docstring.
       Close #1409
 
+    * Check for string concatenation inside lists, sets & tuples, introducing
+      new implicit-str-concatenation-in-literal-iterable (patch by Lucas Cimon)
```

Comment:
Message ID is quite long, maybe we can shorten it up a little bit?

My first thought is something along the lines of `implicit-str-concat-in-sequence`, but I'm open for suggestions.

Reply:
Ok